build.sbt

```diff
@@ -3,32 +3,32 @@ import sbt.dsl.LinterLevel.Ignore
 
 // Library versions all in one place, for convenience and sanity.
 lazy val catsVersion          = "2.7.0"
-lazy val catsEffectVersion    = "3.3.11"
+lazy val catsEffectVersion    = "3.4.2"
 lazy val circeVersion         = "0.14.2"
-lazy val fs2Version           = "3.2.14"
+lazy val fs2Version           = "3.4.0"
 lazy val h2Version            = "1.4.200"
 lazy val hikariVersion        = "4.0.3" // N.B. Hikari v4 introduces a breaking change via slf4j v2
 lazy val kindProjectorVersion = "0.11.2"
 lazy val postGisVersion       = "2.5.1"
 lazy val postgresVersion      = "42.5.1"
-lazy val refinedVersion       = "0.9.28"
+lazy val refinedVersion       = "0.10.1"
 lazy val scalaCheckVersion    = "1.15.4"
-lazy val scalatestVersion     = "3.2.10"
+lazy val scalatestVersion     = "3.2.14"
 lazy val munitVersion         = "1.0.0-M7"
 lazy val shapelessVersion     = "2.3.9"
 lazy val silencerVersion      = "1.7.1"
 lazy val specs2Version        = "4.19.0"
-lazy val scala212Version      = "2.12.15"
-lazy val scala213Version      = "2.13.8"
-lazy val scala30Version       = "3.1.1"
+lazy val scala212Version      = "2.12.17"
+lazy val scala213Version      = "2.13.10"
+lazy val scala3Version       = "3.2.1"
 lazy val slf4jVersion         = "1.7.36"
 lazy val weaverVersion        = "0.7.15"
 
 // Basic versioning and publishing stuff
 ThisBuild / tlBaseVersion := "1.0"
 ThisBuild / tlCiReleaseBranches := Seq("main") // publish snapshits on `main`
-ThisBuild / scalaVersion := scala213Version
-ThisBuild / crossScalaVersions := Seq(scala212Version, scala213Version, scala30Version)
+ThisBuild / scalaVersion := scala212Version
```

Comment:
Why change to 2.12 as the default?

Reply:
Ah was checking some errors in 2.12. Will revert thanks!